free-programming-books.md

```diff
@@ -1689,6 +1689,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Angular for the jQuery developer](http://www.ng-newsletter.com.s3-website-us-east-1.amazonaws.com/posts/angular-for-the-jquery-developer.html)
 * [Angular.js Guide](https://docs.angularjs.org/guide/)
 * [Angular.js Material Designing](https://material.angularjs.org/latest/)
+* [Angular.js Style Guide](https://github.com/toddmotto/angularjs-styleguide)
```

Comment:
Could you please add the author's name after this?
This will also make it more consistent with other github.com links. 

```suggestion
* [Angular.js Style Guide](https://github.com/toddmotto/angularjs-styleguide) - Todd Motto
```